misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JDIEventProcessor.java

```diff
@@ -105,15 +109,27 @@ private boolean processEvent(EventSet eventSet, Event event) {
             context.getClient().stopped(stoppedEventArguments);
             context.getEventManager().deleteEventRequests(stepEventRequests);
         } else if (event instanceof StepEvent) {
-            if (((StepEvent) event).location().lineNumber() > 0) {
-                context.getEventManager().deleteEventRequests(stepEventRequests);
+            StepEvent e = (StepEvent) event;
```

Comment:
Shall we have a meaningful name here instead of `e` for better readability?

Reply:
+1. Added with c1f703a